bricks-maven-plugin/src/main/java/com/github/ingogriebsch/bricks/maven/plugin/analyzer/standard/BasicProjectAnalyzer.java

```diff
@@ -0,0 +1,55 @@
+/*-
+ * #%L
+ * Bricks Maven Plugin
+ * %%
+ * Copyright (C) 2018 - 2019 Ingo Griebsch
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.github.ingogriebsch.bricks.maven.plugin.analyzer.standard;
+
+import com.github.ingogriebsch.bricks.maven.plugin.analyzer.AbstractMavenAnalyzer;
+import com.github.ingogriebsch.bricks.maven.plugin.analyzer.AnalysisResult;
+import com.github.ingogriebsch.bricks.model.Component;
+import com.github.ingogriebsch.bricks.model.MavenCoordinates;
+
+import org.apache.maven.project.MavenProject;
+
+public class BasicProjectAnalyzer extends AbstractMavenAnalyzer {
+
+    @Override
+    protected AnalysisResult augment(Component c) {
+
+        c.setName(project.getName());
+        c.setDescription(project.getDescription());
+
+        c.setMavenCoordinates(fromCoordinates(project));
```

Comment:
As already mentioned, would like to avoid to include such concrete named data. We have more than enough components which have nothing to do with maven.

Reply:
it is optional, and it identifies the exact artifact used *if* component is a maven component.
but hey, feel free to move it to some gloomy corner, as long as you don't loose the info :D